ios/RCTWKWebView/RCTWKWebView.m

```diff
@@ -299,11 +300,67 @@ - (void)stopLoading
   [_webView stopLoading];
 }
 
+- (NSString *) cookieDescription:(NSHTTPCookie *)cookie {
+
+    NSMutableString *cDesc = [[NSMutableString alloc] init];
+    [cDesc appendFormat:@"%@=%@;",
+     [[cookie name] stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding],
+     [[cookie value] stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding]];
+    if ([cookie.domain length] > 0)
+        [cDesc appendFormat:@"domain=%@;", [cookie domain]];
+    if ([cookie.path length] > 0)
+        [cDesc appendFormat:@"path=%@;", [cookie path]];
+    if (cookie.expiresDate != nil)
```

Comment:
Its missing HttpOnly, secure and sessionOnly property. Is this something you would be able to add?

Reply:
for the HttpOnly: I can do that,
for the sessionOnly there ist no such thing, If there is no expire date than it should be a session cookie, otherwise it will be stored until the expire date

Reply:
https://developer.apple.com/documentation/foundation/nshttpcookie/1392991-sessiononly?language=objc

I was referencing that boolean to indicate its a session only cookie

Reply:
And Thanks for fixing this issue!